homework/src/index.js

```diff
@@ -32,12 +32,108 @@
 
   function main(url) {
     fetchJSON(url, (err, data) => {
+
       const root = document.getElementById('root');
+      const header = createAndAppend('header', root, { 'html': '<h3> HYF Repositories <h3>' })
+
+
+      const select = createAndAppend('select', header)
+      const div = createAndAppend('div', root, { 'id': 'div1' })
+      const div2 = createAndAppend('div', root, { 'id': 'div2' })
+
+
       if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
+        createAndAppend('p', div2, { html: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        data.forEach((repo, index) => {
+          const option = createAndAppend('option', select, { 'html': repo.name, 'value': index });
```

Comment:
1. The `option` variable is unused, unneeded and therefore should not be defined here.
2. You do not need quotes around property names if those names adhere to the allowed syntax for JavaScript variable names (e.g. no spaces or special characters). So: `{ html: repo.name, value: index}` would do fine.